promisasu/Portal
# Us#558 & Us#572

**Taiga User Stories**
US#558 : As a clinician, I want a confirmation when I create a patient 
US#572 : As a clinician, I want to know what filter I am using while viewing tables

**Taiga Task(s)**
#605 Create popup screen on new patient create event
#606 Display PIN on popup screen

#607 Fix filter drop down list

**What did you change?**
US#558
1. Created new dialog view named 'remember-patient-pin'
2. Capture new patient enrolment event
3. pop-up dialog to show necessary details on enrolment

US#572
1. Fixed drop down list problem with jQuery

**How can we test?**
US#558
1. Enrol new patient and after enrolment dialog will appear as per attached shot
2. No dialog for already created patient 
![screen shot 2016-04-08 at 4 32 17 pm](https://cloud.githubusercontent.com/assets/5720497/14400024/c12b9384-fda7-11e5-95dc-402d20ecd22b.png)

US#572
1. Trial to use filter on dashboard view.
2. Selected value will appear on selection in filter
![screen shot 2016-04-08 at 4 40 30 pm](https://cloud.githubusercontent.com/assets/5720497/14400120/aa2ca3ca-fda8-11e5-92a9-7d6ad8578974.png)
